guides/models/creating-updating-and-deleting-records.md

```diff
@@ -1,71 +1,59 @@
-## Creating Records
+## Création de _Records_
 
-You can create records by calling the
-[`createRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/createRecord?anchor=createRecord)
-method on the store.
+Vous pouvez créer des _records_ (enregistrements) en appelant la méthode
+[`createRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/createRecord?anchor=createRecord) sur le _store_ (magasin, réserve).
 
 ```javascript
 this.store.createRecord('post', {
   title: 'Rails is Omakase',
-  body: 'Lorem ipsum'
+  body: 'Lorem ipsum',
 });
 ```
 
-To make `store` available, you can [inject the `store` service](../#toc_injecting-the-store).
+Pour rendre disponible le _store_, vous pouvez [injecter le service _`store`_](../#toc_injecting-the-store).
```

Comment:
Note that English uses _a lot_ of "you can" "you can" "you can" "you can" "you can"... which can make the French phrasing a bit heavy sometimes. In French, "pouvoir" is really about suggesting something, so I advise to avoid it from time to time especially when we are just explaining the common way to do something: 

```suggestion
Pour rendre disponible le _store_, il faut [injecter le service _`store`_](../#toc_injecting-the-store).
```